spring-batch-core/src/main/java/org/springframework/batch/core/scope/context/StepContext.java

```diff
@@ -160,7 +160,7 @@ public Map<String, Object> getJobParameters() {
 
 	@SuppressWarnings({"rawtypes", "unchecked"})
 	public Map<String, Object> getPartitionPlan() {
```

Comment:
This method was removed in e5c752b4b82057a80ed2a049770223201c309666, I will omit this change and its related test on merge.